test/StudentRegistryV2.js

```diff
@@ -122,6 +122,17 @@ describe("StudentRegistryV2 Test Suite", () => {
           // Check that the contract's balance increased by 1 ETH
           expect(finalContractBalanceNum).to.be.closeTo(initialContractBalanceNum + 1, 0.01); // Use a tolerance for floating point comparison
         });
+        describe.only("Event", () => {
+          it("should emit PaidFee", async () => {
+            const { deployedStudentRegistryV2, addr1, deployedStudentRegistryV2Address, owner } = await loadFixture(
```

Comment:
only extract variables that you use in your test:
`const { deployedStudentRegistryV2, addr1 } = await loadFixture(deployUtil);`